diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59a3799..1260b2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1114,6 +1114,12 @@ set(LOVE_SRC_MODULE_WINDOW
 source_group("modules\\window" FILES ${LOVE_SRC_MODULE_WINDOW_ROOT})
 source_group("modules\\window\\sdl" FILES ${LOVE_SRC_MODULE_WINDOW_SDL})
 
+set(LOVE_SRC_MODULE_FETCH
+	src/modules/fetch/wrap_Fetch.cpp
+	src/modules/fetch/wrap_Fetch.h
+)
+source_group("modules\\fetch" FILES ${LOVE_SRC_MODULE_FETCH})
+
 ###################################
 # Third-party libraries
 ###################################
@@ -1741,6 +1747,7 @@ set(LOVE_LIB_SRC
 	${LOVE_SRC_MODULE_TOUCH}
 	${LOVE_SRC_MODULE_VIDEO}
 	${LOVE_SRC_MODULE_WINDOW}
+	${LOVE_SRC_MODULE_FETCH}
 )
 
 include_directories(
@@ -1803,6 +1810,7 @@ if(EMSCRIPTEN)
 		"-s FORCE_FILESYSTEM=1" # to fix Module.addDependency not existing
 		"-lidbfs.js" # to fix the ReferenceError: IDBFS is not defined
         "-lwebsocket.js" # websocket API
+		"-sFETCH=1"      # emscripten_fetch() for HTTPS requests from Lua
 	)
 
 	if(NOT LOVEJS_COMPAT)
