--- src/modules/love/love.cpp.orig	2026-08-16 15:43:24.595738039 +0000
+++ src/modules/love/love.cpp	2026-08-16 15:43:42.707004733 +0000
@@ -163,6 +163,7 @@
 #if defined(LOVE_ENABLE_WINDOW)
 	extern int luaopen_love_window(lua_State*);
 #endif
+	extern int luaopen_love_fetch(lua_State*);
 	extern int luaopen_love_nogame(lua_State*);
 	extern int luaopen_love_jitsetup(lua_State*);
 	extern int luaopen_love_arg(lua_State*);
@@ -228,6 +229,7 @@
 #if defined(LOVE_ENABLE_WINDOW)
 	{ "love.window", luaopen_love_window },
 #endif
+	{ "love.fetch", luaopen_love_fetch },
 	{ "love.nogame", luaopen_love_nogame },
 	{ "love.jitsetup", luaopen_love_jitsetup },
 	{ "love.arg", luaopen_love_arg },
